xtevent/xtevent.sthlp

```diff
@@ -44,6 +44,7 @@
 {synopt:{opt note}} omit time fixed effects {p_end}
 {synopt: {opt impute(type, [saveimp])}} impute missing values in policyvar{p_end}
 {synopt:{opt st:atic}} estimate static model {p_end}
+{synopt:{opt diffavg}} estimate the regular DiD coefficient and standard error implied by the model {p_end}

```

Comment:
@rayhuang11 do not use DiD coefficient terminology, just talk about differences in averages.